api/tests/routes/pro/get_stocks_test.py

```diff
@@ -0,0 +1,155 @@
+from datetime import datetime
+from datetime import timedelta
+
+from freezegun import freeze_time
+import pytest
+
+import pcapi.core.offerers.factories as offerers_factories
+import pcapi.core.offers.factories as offers_factories
+import pcapi.core.users.factories as users_factories
+
+from tests.conftest import TestClient
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns403Test:
+    def test_access_by_beneficiary(self, app):
+        # Given
+        beneficiary = users_factories.BeneficiaryGrant18Factory()
+        offer = offers_factories.ThingOfferFactory(venue__latitude=None, venue__longitude=None)
+
+        # When
+        client = TestClient(app.test_client()).with_session_auth(email=beneficiary.email)
+        response = client.get(f"/offers/{offer.id}/stocks")
+
+        # Then
+        assert response.status_code == 403
+
+    def test_access_by_unauthorized_pro_user(self, app):
+        # Given
+        pro_user = users_factories.ProFactory()
+        offer = offers_factories.ThingOfferFactory(venue__latitude=None, venue__longitude=None)
+
+        # When
+        client = TestClient(app.test_client()).with_session_auth(email=pro_user.email)
+        response = client.get(f"/offers/{offer.id}/stocks")
+
+        # Then
+        assert response.status_code == 403
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns200Test:
+    def test_basic(self, app):
+        # Given
+        user_offerer = offerers_factories.UserOffererFactory()
+        offer = offers_factories.ThingOfferFactory(venue__managingOfferer=user_offerer.offerer)
+
+        # When
+        client = TestClient(app.test_client()).with_session_auth(email=user_offerer.user.email)
+        response = client.get(f"/offers/{offer.id}/stocks/")
+
+        # Then
+        assert response.status_code == 200
+
+    @freeze_time("2020-10-15 00:00:00")
+    def test_returns_an_event_stock(self, app):
+        # Given
+        now = datetime.utcnow()
```

Comment:
idem dans tout ce fichier, `@freeze_time` laisse penser qu'il y a un appel à `datetime.utcnow()` dans le code testé alors que ce n'est pas le cas

```suggestion
    def test_returns_an_event_stock(self, app):
        # Given
        now = datetime.datetime(2020, 10, 15)
```

Reply:
En fait là si, car la création de Stock via EventStockFactory va comparer la limite de booking avec datetime.datetime.now() et donc renvoyer isBookable = False si on n'utilise pas @freeze-time (contrairement à ce qu'on demande par la suite dans le test)